python/tvm/relay/frontend/tflite.py

```diff
@@ -1770,6 +1775,12 @@ def convert_pool2d(self, op, pool_type):
                 assert self.has_same_qnn_params(input_tensor, output_tensor), \
                         "qnn.op.max_pool2d requires input and output qnn params to be same"
             out = _op.nn.max_pool2d(in_expr, **params)
+        elif pool_type == "l2":
+            # l2_pool_2d is equivalent to square_root(avg_pool(square(in_data)))
```

Comment:
Add a protection check to throw error if you receive quantized inputs.

Reply:
TFLite does not support quantized L2_POOL_2D. That's why the check is not added. Now I have added a comment for the same.

Reply:
Putting in the check has the following benefits:

1. Better user experience as the rest of the stack doesn't get basically undefined behaviour because of random input.. 
2. A signal to us as a community in the form of a bug report that this is appearing. 

Ramana

Reply:
ok, make sense. I have added the check.